### PR TITLE
Deploy to vm

### PR DESCRIPTION
**Pull request by danzvara**
Some scripts

### CODE REVIEW

File: deploy-vm.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        exit 1
+    fi
+}
+
+check_dep pwgen
+
+HOME=$(pwd)
+PASS=$(pwgen 10 1)
+cd $HOME/parity && echo $PASS > password.txt
+ACCOUNT=$(parity --chain ./res/genesis.json --keys-path ./datadir//keys account new --password password.txt)
```
Comment (fangyi-zhou) [inline deploy-vm.sh]: Don't you have to call bank here?

Comment (levex) [reply deploy-vm.sh]: Yes

File: deploy-vm.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+check_dep () {
+    if ! [ -x "$(command -v $1)" ]; then
+        echo "You don't have $1"
+        exit 1
+    fi
+}
+
+check_dep pwgen
+
+HOME=$(pwd)
```
Comment (levex) [inline deploy-vm.sh]: can you rename this... so confusing

Comment (fangyi-zhou) [reply deploy-vm.sh]: rename what?

**Pull request merged**